docs/reference/query-dsl/geo-bounding-box-query.asciidoc

```diff
@@ -188,10 +188,10 @@ values separately.
         "filter" : {
             "geo_bounding_box" : {
                 "pin.location" : {
-                    "top" : -74.1,
-                    "left" : 40.73,
-                    "bottom" : -71.12,
-                    "right" : 40.01
+                    "top" : 40.73,
```

Comment:
I agree that this looks like the values were switched... but shouldn't it work anyway? It's just that the bounding box would cross both poles.  @nknize ?